src/ring_theory/discrete_valuation_ring.lean

```diff
@@ -90,6 +91,10 @@ begin
     use a * b, exact hh.symm }
 end⟩
 
+lemma _root_.irreducible.maximal_ideal_eq {ϖ : R} (h : irreducible ϖ) :
```

Comment:
```suggestion
lemma _root_.irreducible.maximal_ideal_eq_of_irreducible {ϖ : R} (h : irreducible ϖ) :
```
Just a suggestion... it's more verbose, but perhaps easier to find later?

Reply:
But it's already in the `irreducible` namespace...

Reply:
Oh of course. I'm being stupid. Ignore!